tests/test_pagure.py

```diff
@@ -0,0 +1,749 @@
+        self.assertEqual(issue['data']['assignee']['name'], 'animeshk08')
+        self.assertEqual(len(issue['data']['comments'][0]['reactions']), 0)
+
+        issue = issues[1]
+        self.assertEqual(issue['origin'], 'https://pagure.io/Project-example')
+        self.assertEqual(issue['uuid'], '7dd3642664c8a7e475814b9037277df775657850')
+        self.assertEqual(issue['updated_on'], 1583558174.0)
+        self.assertEqual(issue['category'], CATEGORY_ISSUE)
+        self.assertEqual(issue['tag'], 'https://pagure.io/Project-example')
+        self.assertEqual(issue['data']['assignee']['name'], 'animeshk0806')
+        self.assertEqual(len(issue['data']['comments']), 2)
+        self.assertEqual(issue['data']['comments'][0]['user']['name'], 'animeshk08')
+        self.assertEqual(len(issue['data']['comments'][0]['reactions']), 0)
+        self.assertEqual(len(issue['data']['comments'][1]['reactions']), 1)
+        self.assertListEqual(issue['data']['comments'][1]['reactions']['Thumbs up'], ['animeshk0806'])
+
+    @httpretty.activate
+    def test_fetch_more_issues(self):
+        """Test when return two issues"""
+
+        issue_2 = read_file('data/pagure/pagure_repo_issue_2')
+
+        httpretty.register_uri(httpretty.GET,
+                               PAGURE_ISSUES_URL,
+                               body=issue_2,
+                               status=200,
+                               )
+
+        pagure = Pagure(repository='Project-example')
+        issues = [issues for issues in pagure.fetch()]
+
+        self.assertEqual(len(issues), 2)
+
+        issue = issues[0]
+        self.assertEqual(issue['origin'], 'https://pagure.io/Project-example')
+        self.assertEqual(issue['uuid'], '41071b08dd75f34ca92c6d5ecb844e7a3e5939c6')
+        self.assertEqual(issue['updated_on'], 1583508642.0)
+        self.assertEqual(issue['category'], CATEGORY_ISSUE)
+        self.assertEqual(issue['tag'], 'https://pagure.io/Project-example')
+        self.assertEqual(len(issue['data']['comments']), 1)
+        self.assertEqual(issue['data']['comments'][0]['user']['name'], 'animeshk08')
+        self.assertEqual(issue['data']['assignee']['name'], 'animeshk08')
+
+        issue = issues[1]
+        self.assertEqual(issue['origin'], 'https://pagure.io/Project-example')
+        self.assertEqual(issue['uuid'], '7dd3642664c8a7e475814b9037277df775657850')
+        self.assertEqual(issue['updated_on'], 1583558174.0)
+        self.assertEqual(issue['category'], CATEGORY_ISSUE)
+        self.assertEqual(issue['tag'], 'https://pagure.io/Project-example')
+        self.assertEqual(issue['data']['assignee']['name'], 'animeshk0806')
+        self.assertEqual(len(issue['data']['comments']), 2)
+        self.assertEqual(issue['data']['comments'][0]['user']['name'], 'animeshk08')
+
+    @httpretty.activate
+    def test_fetch_issues_until_date(self):
+        """Test when return issue till a particular date"""
```

Comment:
The test data should include two issues, one outside the to_date (which should be the one returned)